js/index.js

```diff
@@ -0,0 +1,22 @@
+window.addEventListener('DOMContentLoaded', () => {
+    const btn = document.querySelector('#menu_btn');
+    console.log(btn);
+    const menu = document.querySelector('#mobile-nav');
+    console.log(menu);
+    const close = document.querySelector('#mobile-nav span');
+    console.log(close);
+    const list = document.querySelectorAll('#mobile-nav ul li');
+    // console.log(list);
+  
+    btn.addEventListener('click', () => {
+        menu.style.display = 'flex';
+    })
+    close.addEventListener('click', () => {
+      menu.style.display = 'nonne';
+    })
```

Comment:
- [ ] In the project requirements, When the user clicks (or taps) the close (X) button, the mobile menu disappears. But your close button is not working as expected because when you click it nothing happens. You should make it work as expected . This will improve your project which will make you look more professional in eyes of hiring managers, etc. You can do it by correcting the typo in line 15, By just writing `none` instead of `nonne`